src/mnist_loader.py

```diff
@@ -112,9 +143,12 @@ def _load_label(filename):
     return data
 
 
+
+
 def main():
     """Totally useless main."""
     (train_iamges, train_labels, t10k_images, t10k_labels) = load_mnist()
+    (train_iamges, train_labels, val_images, val_labels, test_imgages, test_labels) = load_mnist(validation_proportion=0.1)
```

Comment:
Is this going to break existing calls to mnist_loader? As in, defaulting it to return a validation set will return two extra arguments that certain training scripts are not currently ready to handle. Would you mind updating the scripts or have a way to pass to this the validation_proportion so that it doesn't return the validation set by accident?

Reply:
Nope! I can't decide which is better
1.  Returning validation stuff as (like train_img, train_label, test_img test_label, val_img, val_label), which is a little weird but doesn't change the first four args if no val set is requested , or
2. Returning them in the normal order (train_img, train_label, val_img, val_label, test_img test_label) or  (train_img, train_label, test_img test_label), depending. 

Neither should break anything. I had #2, but switched it to #1.